app/helpers/template_helpers.py

```diff
@@ -61,7 +61,12 @@ def context(self) -> dict[str, Any]:
             "account_service_todo_url": self._survey_config.account_service_todo_url,
             "contact_us_url": self._survey_config.contact_us_url,
             "thank_you_url": url_for("post_submission.get_thank_you"),
-            "cookie_settings_url": self._survey_config.cookie_settings_url,
+            "cookie_settings_url": self._survey_config.cookie_settings_url
+            if cookie_session
+            else None,
+            "cookie_address": self._survey_config.cookie_address
+            if cookie_session
+            else None,
```

Comment:
We should add these to the context conditionally instead of inline where the key always exist. Set it if we have something to set.

Reply:
Adding conditionally now, had to make some tweaks but working now.